qiskit_nature/second_q/properties/__init__.py

```diff
@@ -0,0 +1,65 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Second-Quantization Properties (:mod:`qiskit_nature.second_q.properties`)
+================================================================================================
+
+.. currentmodule:: qiskit_nature.second_q.properties
+
+"""
+
+from .grouped_property import GroupedProperty
+from .property import Property
+from .second_quantized_property import GroupedSecondQuantizedProperty, SecondQuantizedProperty
+
+from .fermi_hubbard_model import FermiHubbardModel
+from .ising_model import IsingModel
+from .lattice_model import LatticeModel
+
+from .lattices.boundary_condition import BoundaryCondition
+from .lattices.hyper_cubic_lattice import HyperCubicLattice
+from .lattices.lattice import LatticeDrawStyle, Lattice
+from .lattices.line_lattice import LineLattice
+from .lattices.square_lattice import SquareLattice
+from .lattices.triangular_lattice import TriangularLattice
+
+from .angular_momentum import AngularMomentum
+from .dipole_moment import DipoleMoment, ElectronicDipoleMoment
+from .electronic_structure_driver_result import ElectronicStructureDriverResult
+from .electronic_energy import ElectronicEnergy
+from .magnetization import Magnetization
+from .particle_number import ParticleNumber
+
+from .occupied_modals import OccupiedModals
+from .vibrational_structure_driver_result import VibrationalStructureDriverResult
+from .vibrational_energy import VibrationalEnergy
+
+__all__ = [
```

Comment:
There are more imports here than in the list below. Maybe thats fine. If you do an `from qiskit_nature.second_q.properties import *` you get what is in all. This will not include LineLattice for instance. I guess I like to try and keep the all list in the same order as the imports for easier checking.